### ISSUE TITLE
Wider FOV with motorized mount?

### ISSUE THREAD

**Issue by jzellis** [alextousss/3d_ultrasonic_scanner#2]
Hi! First of all, this is amazing, well done!

You mentioned in your blog that it's limited to about a 45° FOV in two axes. Could you mount it on a stepper motor to rotate the transducers in one or both axes and then "stitch" your captured plots together, in the way that a phone camera stitches panoramic shots? It might not be useful in an environment with moving objects - or if the scanner is moving itself - but it would probably provide a cheap and easy way to extend the capabilities without necessarily building more arrays.

(The motor itself might produce noise, of course, but that could probably be compensated for.)

Dunno, just a thought. But keep up the good work! If you get to the point of producing a kit or commercial product, I'd love to get one and try it out!

**Comment by alextousss** [comment]
Yes, you could mount it on steppers to have a 360° FOV.  Unless the stepper's hashing frequency is close to 40khz, the motor wouldn't interfere at all with the scanner!

I don't think I'll sell a kit until next year. In the meantime, I can definitely build a few  units of my current design. It takes me quite some time to build one though. If it isn't too rude to ask: how much would you pay to get an unit?

Thank you for the compliments!
